### DISCUSSION TITLE
External Editor doesn't launch from shortcut

### DISCUSSION THREAD

**Discussion by meteorquake** [WinMerge/winmerge#2150] [Ideas]
If I right-click and launch an item with External Editor it works fine.
But if I use Ctrl-Alt-E (the shortcut listed against it) nothing happens.
2.16.36.0 x86 - W10 64-bit
Cheers, David

**Comment by sdottaka** [comment]
I couldn't reproduce this issue. Do you have any software that registers Ctrl+Alt+E as a global shortcut key?

**Comment by meteorquake** [comment]
Thanks for looking - I don't seem to have a conflicting shortcut, and I closed Foobar2000 (particularly) and Everything just to be sure and still no luck. I don't think I've ever seen it work on prior versions. I've also just tried it on another computer and the same issue occurs. I also tried the on screen keyboard (OSK) and as I have a Turkish Querty I also added an English layout and tried that, also trying OSK, all with no luck...
David
